idnc/estimation/test_fullBox.txt

```diff
@@ -0,0 +1,5 @@
+1 0.5 0.5 1 1
+0 0.870313 0.540625 0.071875 0.09375
+0 0.03125 0.548958 0.0625 0.0895833
+0 0.242969 0.35 0.0390625 0.05
+0 0.710156 0.340625 0.0359375 0.04375

```

Comment:
Duplicate of  [tests/estimation/test_fullBox.txt](https://github.com/GaboFlo/IDNConsensus/pull/8/files?short_path=1b067e6#diff-cc23ba037fcc4e7a2614700ae99750969852c747aa35f9bb55f5da30426f5231)